src/js/select2/i18n/ka.js

```diff
@@ -0,0 +1,38 @@
+define(function () {
+  // Georgian
+  return {
+    errorLoading: function () {
+      return 'სია ცარიელია.';
```

Comment:
I don't have anyone to reach out to about this translation, and the automated translators are giving me something along the lines of "To display".